### PR TITLE
Chore: Remove ANSI color codes from perfetto logs

### PR DESCRIPTION
**Pull request by apbodnar**
Improves log readability by removing incorrectly rendered ANSI color codes from perfetto stdout/stderr

### CODE REVIEW

File: core/os/android/adb/perfetto.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ import (
 )
 
 var (
-	ansiRegex = regexp.MustCompile("\u001b\\[\\d+m")
+	AnsiRegex = regexp.MustCompile("\u001b\\[\\d+m")
```
Comment (AWoloszyn) [inline core/os/android/adb/perfetto.go]: Have we considered filing a bug with perfetto to remove this via a command-line option (or better yet they should be removing them automatically if they detect stdout is not going to shell)

Comment (apbodnar) [reply core/os/android/adb/perfetto.go]: I'll file a bug. My assumption was that we'd still have to work around it for current android version(s).

Comment (apbodnar) [reply core/os/android/adb/perfetto.go]: Internal bug: b/145150184

**Comment by pau-baiget** [comment]
If no other reviewers find issues with the current patch, please merge it :)

**Pull request merged**